content/admin/policy_mgmt/Sensitive-data-masking.md

```diff
@@ -0,0 +1,61 @@
+
+### Agent identification and masking
+
+Contrast agents mask sensitive data in query parameters, request headers, cookies and forms. Your agent identifies sensitive data by searching for specific keywords used in the input name. If the agent finds a match, it redacts the value for that input, and replaces it with a placeholder with the format `contrast-redacted-{datatype}`, where `datatype` is the category of sensitive data to which the keyword belongs.
+
+Contrast agents do **not** mask individual fields in request bodies with a content type other than `application/x-www-form-urlencoded`; however, you can configure the agent to mask the entire request body. Contrast agents also do not mask data that appears in the data flow portion of a vulnerability report, if using Assess, or in the vector of an attack event, if using Protect. 
+
+> **Note:** Contrast agents make a “best effort” attempt to avoid printing sensitive data in debug log statements; however, it’s possible that sensitive data could appear in the Contrast log, if the log level is set to DEBUG or lower. Whenever possible, you should avoid setting production systems to log at DEBUG or lower. If a system that deals with sensitive data is set to log at DEBUG or lower, you should take steps to ensure that those logs are not being sent to an external system to avoid leaking any sensitive data. 
+
+#### Example
+
+The following sample HTTP request sent by an agent as part of a vulnerability report shows two inputs that the agent identified as sensitive as well as the placeholders it used to mask the values of the input before sending the report to the Contrast UI, syslog server or security log. 
+ 
+```
+POST/employee/5 HTTP 1.1 
+
+Host: example.com 
+
+Content-Type: application/x-www-form-urlencoded 
+
+Content-Length: 30 
+
+apikey: contrast-redacted-authentication-info 
+ 
+
+ssn=contrast-redacted-government-id&department=sales 
+
+``` 
+
+In this case, the header value is masked because "apikey" matches a keyword in the "Authentication" data type, and the form parameter is redacted because "ssn" matches a keyword in the "ID" data type for your Contrast organization. (Keyword matches are case insensitive.)
+
+## Manage Sensitive Data Types
+
+Go to the **user menu > Policy Management > Sensitive Data tab** to view and edit the sensitive data types for your organization. In the **Sensitive Data** grid, data types and associated keywords are listed alphabetically. To quickly find data types by name or keywords, use the search field above the grid. 
+
+To enable redaction of the entire HTTP request entity, check the box to **Mask entire body**. This will apply to all applications in your organization.
```

Comment:
I don't think "entire HTTP request entity" is correct here. I think that would mean EVERYTHING, but this is specific to one field, which can be large, but it doesn't cover headers, cookies, etc. Right @seschis?

Reply:
Correct. The entity is the body and related headers (e.g. content-length, etc), so saying entity body, or in this case body would be better. Not sure why I said entity here.